toolbox

```diff
@@ -832,12 +831,21 @@ init_container()
             fi
         fi
 
+        groups=
+        if getent group sudo > /dev/null 2>&3; then
+            groups="sudo"
+        elif getent group wheel > /dev/null 2>&3; then
+            groups="wheel"
+        else
+            echo "$base_toolbox_command: failed to setup groups for user $init_container_user, required group 'sudo' or 'wheel' missing" >&2
+        fi
+
         if ! useradd \
                      --home-dir "$init_container_home" \
                      --no-create-home \
                      --shell "$init_container_shell" \
                      --uid "$init_container_uid" \
-                     --groups wheel \
+                     --groups $groups \
```

Comment:
We should quote the `$groups` to avoid triggering [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086) in future. ShellCheck sometimes misses some things but can start warning later. :/

Reply:
Done.